app/controllers/sessions_controller.rb

```diff
@@ -1,6 +1,9 @@
 class SessionsController < ApplicationController
   def create
-    session["username"] = auth_info["uid"]
+    session["user"] = {
+      email: auth_info["uid"],
```

Comment:
Suggestion:  I'm not sure this is the email, but rather is the username?  Would you mind switching this back or adding username and email? This is what the debug line below outputs:

```
"uid": "test.rosps",
  "info": {
    "name": "Test ID RO Supervisory Program Specialist",
    "email": "33212@hsesinfo.org",

```

Reply:
Relatedly, the username field in HSES is often, but not always, actually an email address.

Reply:
I'd be happy to change the field to `uid`; I'd rather not have a name and a username because that's confusing to me. (I'm also curious if, in reality, the uid is ever not the user's email address?)

On reflection, I don't think we currently need their email anyways; if we ever get into sending notifications we will, but currently a ways away from that.

Reply:
> (I'm also curious if, in reality, the uid is ever not the user's email address?)

In production, yes, the uid is often not an email address. See: https://github.com/HHS/Head-Start-TTADP/blob/ce6f7d5a22e51949de680c31a644c9bb9dcb01b6/src/app.js#L80-L83 for how TTA Hub assigns the user email address to their saved HSES users some of the time.

Reply:
cool, thanks for those details!